clients/instance/ibm-pi-snapshot.go

```diff
@@ -81,8 +81,8 @@ func (f *IBMPISnapshotClient) GetAll() (*models.Snapshots, error) {
 	return resp.Payload, nil
 }
 
-// Create or Restore a Snapshot
-func (f *IBMPISnapshotClient) Create(instanceID, snapshotID, restoreFailAction string) (*models.Snapshot, error) {
+// Restore a Snapshot
+func (f *IBMPISnapshotClient) Restore(instanceID, snapshotID, restoreFailAction string) (*models.Snapshot, error) {
```

Comment:
This is a breaking change. You can create another function Restore and call Create(). Also, you could mark it as deprecated.

Reply:
@yussufsh How do I mark elements in the power-go-client as deprecated?